app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -36,8 +36,40 @@ const SendValidation = ({sendFormValidationError, sendResponse}) =>
       sendResponse.success &&
       h('div', {class: 'validation-message transaction-success'}, 'Transaction successful!')
 
-class SendAdaPage extends Component {
-  // Needs to be a component because we keep refs
+
+interface Props {
```

Comment:
Looking at this component, I suspect it has too much responsibility! And even if not, maybe we should probably structure some props as a single object